Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1280,7 +1280,7 @@ def is_bot_message(data: dict) -> bool:
         return True
     elif event.get('subtype') == 'bot_message':
         return True
-    elif data.get('authorizations', [{}])[0].get('is_bot', False):
+    elif not (event.get('user') or data.get('user', {}).get('id') or data.get('envelope_id')):
```

Comment:
Looks good. One thing we can also do is to make sure that the User string doesn't start with "B". For bots, a user ID is always in the format BXXX123. It's just another little check we can do.